src/vs/workbench/api/node/extHostApiCommands.ts

```diff
@@ -240,6 +240,15 @@ export class ExtHostApiCommands {
 					{ name: 'column', description: '(optional) Column in which to open', constraint: v => v === void 0 || typeof v === 'number' }
 				]
 			});
+
+		this._register('vscode.pickWorkspace', (options?: vscode.QuickPickOptions) => {
+			return this._commands.executeCommand('_workbench.pickWorkspace', [options]);
```

Comment:
What does this return? It must be triple-equal to the things we have in `workspace.workspaceFolders`. Also, why not have this as real API? Having the logic in the core shouldn't mean we cannot have a function for this